docs/B-Computations/types.md

```diff
@@ -13,476 +17,512 @@ After reading this section, you will be able to:
 
 A **`char`** occupies one byte and can store a small integer value, a single character or a single symbol: 
-
-<table border="0">
-<tr><td align="center" colspan="8">char</td></tr>
-<tr><td align="center" colspan="8">1 Byte</td></tr>
-<tr><td bgcolor="#ffffcc">&nbsp;</td>
-    <td bgcolor="#ffffcc">&nbsp;</td>
-    <td bgcolor="#ffffcc">&nbsp;</td>
-    <td bgcolor="#ffffcc">&nbsp;</td>
-    <td bgcolor="#ffffcc">&nbsp;</td>
-    <td bgcolor="#ffffcc">&nbsp;</td>
-    <td bgcolor="#ffffcc">&nbsp;</td>
-    <td bgcolor="#ffffcc">&nbsp;</td></tr>
-</table>
-
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 960 160">
+    <foreignObject x="36%" width="110%" height="110%">
+        <body xmlns="http://www.w3.org/1999/xhtml">
+            <table border="0">
+            <tr><td align="center" colspan="8"><code>char</code></td></tr>
+            <tr><td align="center" colspan="8">1 Byte</td></tr>
+            <tr><td>&nbsp;</td>
+                <td>&nbsp;</td>
+                <td>&nbsp;</td>
+                <td>&nbsp;</td>
+                <td>&nbsp;</td>
+                <td>&nbsp;</td>
+                <td>&nbsp;</td>
+                <td>&nbsp;</td></tr>
+            </table>
+        </body>
+    </foreignObject>
+</svg>
 
 An **`int`** occupies one word and can store an integer value. In a 32-bit environment, an **`int`** occupies 4 bytes:
-
-<table border="0">
-<tr><td align="center" colspan="32">int (32-bit environment)</td></tr>
-
-<tr><td align="center" colspan="8">1 Byte</td>
-    <td align="center" colspan="8">1 Byte</td>
-    <td align="center" colspan="8">1 Byte</td>
-    <td align="center" colspan="8">1 Byte</td></tr>
-
-<tr><td bgcolor="#ffffcc">&nbsp;</td>
-    <td bgcolor="#ffffcc">&nbsp;</td>
-    <td bgcolor="#ffffcc">&nbsp;</td>
-    <td bgcolor="#ffffcc">&nbsp;</td>
-    <td bgcolor="#ffffcc">&nbsp;</td>
-    <td bgcolor="#ffffcc">&nbsp;</td>
-    <td bgcolor="#ffffcc">&nbsp;</td>
-    <td bgcolor="#ffffcc">&nbsp;</td>
-    <td bgcolor="#ffffcc">&nbsp;</td>
-    <td bgcolor="#ffffcc">&nbsp;</td>
-    <td bgcolor="#ffffcc">&nbsp;</td>
-    <td bgcolor="#ffffcc">&nbsp;</td>
-    <td bgcolor="#ffffcc">&nbsp;</td>
-    <td bgcolor="#ffffcc">&nbsp;</td>
-    <td bgcolor="#ffffcc">&nbsp;</td>
-    <td bgcolor="#ffffcc">&nbsp;</td>
-    <td bgcolor="#ffffcc">&nbsp;</td>
-    <td bgcolor="#ffffcc">&nbsp;</td>
-    <td bgcolor="#ffffcc">&nbsp;</td>
-    <td bgcolor="#ffffcc">&nbsp;</td>
-    <td bgcolor="#ffffcc">&nbsp;</td>
-    <td bgcolor="#ffffcc">&nbsp;</td>
-    <td bgcolor="#ffffcc">&nbsp;</td>
-    <td bgcolor="#ffffcc">&nbsp;</td>
-    <td bgcolor="#ffffcc">&nbsp;</td>
-    <td bgcolor="#ffffcc">&nbsp;</td>
-    <td bgcolor="#ffffcc">&nbsp;</td>
-    <td bgcolor="#ffffcc">&nbsp;</td>
-    <td bgcolor="#ffffcc">&nbsp;</td>
-    <td bgcolor="#ffffcc">&nbsp;</td>
-    <td bgcolor="#ffffcc">&nbsp;</td>
-    <td bgcolor="#ffffcc">&nbsp;</td></tr>
-</table>
-
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 980 165">
```

Comment:
With these super wide tables, you need to constrain the `max-width` so that the SVG table can't go so wide that it overflows.